src/main/java/com/google/sps/data/SiteDetails.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/** 
+ * This class represents the site details provided by NSW GOV API for each site avaialble for fitering.
+ */
+public class SiteDetails {
```

Comment:
Depending on what the other APIs are like (i.e. if they need a similar process of getting the site details), it could make it less confusing later on to have NSW Gov somewhere in the class name